src/gennylib/src/encryption.cpp

```diff
@@ -305,6 +332,55 @@ void FLEEncryptedField::appendEncryptInfo(sub_document subdoc) const {
     }
 }
 
+QueryableEncryptedField::Query::Query(const Node& yaml) {
+    queryType = yaml["queryType"].to<std::string>();
+    contention = yaml["contention"].maybe<int>();
```

Comment:
Should the default be explicit here?

Reply:
I do not want to hard-code an explicit contention default value here as it's probably going to change again in the near future.